src/components/ActivityFeed/ActivityFeedPostHeader.tsx

```
@@ -98,7 +98,7 @@ export const ActivityFeedPostHeader: React.FC<IPostHeaderProps> = ({
 		const totalVotes = ayesNumber + naysNumber;
 		const ayesPercentage = totalVotes > 0 ? (ayesNumber / totalVotes) * 100 : 0;
 		const naysPercentage = totalVotes > 0 ? (naysNumber / totalVotes) * 100 : 0;
-		return { ayesPercentage, isAyeNaN: Number.isNaN(ayesPercentage), isNayNaN: Number.isNaN(naysPercentage), naysPercentage };
+		return { ayesPercentage, isAyeNaN: isNaN(ayesPercentage), isNayNaN: isNaN(naysPercentage), naysPercentage };
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)